vendor.conf

```diff
@@ -26,8 +26,8 @@ github.com/docker/go-units 47565b4f722fb6ceae66b95f853feed578a4a51c # v0.3.3
 github.com/docker/libtrust 9cbd2a1374f46905c68a4eb3694a130610adc62a
 github.com/docker/licensing 9781369abdb5281cdc07a2a446c6df01347ec793
 github.com/docker/swarmkit 415dc72789e2b733ea884f09188c286ca187d8ec
+github.com/evanphx/json-patch 5858425f75500d40c52783dce87d085a483ce135
```

Comment:
I see this package provides tagged releases; do we need the changes that are in master and not yet in their latest release (`v4.1.0`)? https://github.com/evanphx/json-patch/compare/v4.1.0...5858425f75500d40c52783dce87d085a483ce135 

Otherwise, I'd prefer pinning to `v4.1.0` (or ask them to tag a new release)


FWIW this is licensed [BSD3-clause](https://github.com/evanphx/json-patch/blob/master/LICENSE) so we may have to change packaging to show the license? @seemethere @justincormack

Reply:
@thaJeztah the revision for json-patch can from compose-on-kubernetes [gopkg.toml](https://github.com/docker/compose-on-kubernetes/blob/master/Gopkg.toml). I've checked vendoring with v4.1.0 here doesn't break any of the tests. Do you want me to make this change or stick with the revision to match compose-on-kubernetes?

Reply:
If it's ok with you, I'd prefer using a version that matches a tagged release for dependencies that use tags; doing so provides a bit more "context" (also preparing/taking into account users that start using `go mod` for vendoring)

Reply:
sure, done